webapp/src/main/java/com/box/l10n/mojito/metrics/NamingConventionConverter.java

```diff
@@ -0,0 +1,42 @@
+package com.box.l10n.mojito.metrics;
+
+import com.google.common.base.Strings;
+
+import org.springframework.boot.context.properties.ConfigurationPropertiesBinding;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+import io.micrometer.core.instrument.config.NamingConvention;
+
+@Component
+@ConfigurationPropertiesBinding
+public class NamingConventionConverter implements Converter<String, NamingConvention> {
+
+    @Override
+    public NamingConvention convert(String from) {
```

Comment:
I've done something similar using reflection here: https://github.com/box/mojito/commit/1531cd746db5d6e7f1e64f3f0b4a74f42616c22c.

A little more concise and won't need to be extended if the NamingConvention changes